src/pages/Homepage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import logo from '../images/logo.svg';
+import '../styles/Homepage.css';
+
+const Homepage: React.FC = () => {
+
+  const onPlayClick = () => {
+    // TODO: Write code here to redirect to course info screen
+  }
+
+  const onInstructionsClick = () => {
+    // TODO: Write code here to redirect to instructions screen
+  }
+
+  return (
+    <div className="Homepage">
+      <div className="Homepage-Header">
+        <div className="Logo-Container">
+          <img src={logo} className="Logo" alt="logo"/>
+          <p className="Logo-Title">HeaRT</p>
+        </div>
+        <p className="App-Name">Hearing And Realising Teaching-voice</p>
```

Comment:
nitpicking but App-Name for a class is a bit misleading. Would be nice to have something like "app-description" or "app-slogan"